config/main.py

```diff
@@ -889,6 +889,25 @@ def validate_mirror_session_config(config_db, session_name, dst_port, src_port,
 
     return True
 
+def validate_ip_address(ctx, ip_addr):
+    mask_range = 32
+    split_ip_mask = ip_addr.split("/")
+
+    try:
+        ip_addr = str(ipaddress.IPv4Address(split_ip_mask[0]))
+    except ipaddress.AddressValueError:
```

Comment:
This doesn't look good to me (catching as exception for IPv6 address).

Reply:
I agree. Suggest using `ipaddress.ip_address()` and checking whether the type of the result is `IPv4Address` or `IPv6Address`

Reply:
Done